build/README.md

```diff
@@ -29,10 +29,40 @@ Build with different supported versions of Spark, by default using Spark 2.2.1 t
 ```
 mvn -DskipTests -Pspark-2.1 -Dspark.version=2.1.0 clean package
 mvn -DskipTests -Pspark-2.2 -Dspark.version=2.2.1 clean package
+mvn -DskipTests -Pspark-2.3 -Dspark.version=2.3.2 clean package
 ```
 
 Note: If you are working in Windows environment, remember to add `-Pwindows` while building the project.
 
+## MV Feature Build
```

Comment:
no need so lone describe, only add `-Pmv` , for example:If you want to use MV, remember to add `-Pmv ` while building the project.

Reply:
in the parent pom.xml, there is already mv profile.